client/a8c-for-agencies/style.scss

```
@@ -219,16 +219,63 @@
 		border-color: var(--color-primary-60);
```

```suggestion
		border-color: var(--color-primary-50);
```

And it would be great to have same blue in the line above if possible:

```
	background-color: var(--color-primary-50);
```